src/sentry/integrations/gitlab/integration.py

```diff
@@ -22,10 +28,37 @@
 from .repository import GitlabRepositoryProvider
 
 DESCRIPTION = """
-Fill me out
+Connect your Sentry organization to your self-hosted GitLab instance, enabling the following features:
```

Comment:
Perhaps, since we're doing it for both gitlab self hosted and gitlab.com we should say something that is inclusive of both? Or were you thinking of making two integrations?

Reply:
I don't think we'll need two integrations. I'll update the wording.